dev/docker-compose.yml

```diff
@@ -2,14 +2,13 @@ version: "3.9"
 
 services:
   mssql:
-    image: mcr.microsoft.com/mssql/server:2017-latest
+    image: mcr.microsoft.com/azure-sql-edge:latest
     restart: always
     environment:
       ACCEPT_EULA: Y
-      SA_PASSWORD: ${MSSQL_PASSWORD}
+      MSSQL_SA_PASSWORD: ${MSSQL_PASSWORD}
       MSSQL_PID: Developer
     volumes:
-      - mssql_dev_data:/var/opt/mssql/data
```

Comment:
Where would the new local data file should be located? Is there a volume mapped or would it be wholly self-contained in the running container (ephemeral to that instance)?

Reply:
Mounting a volume is currently not supported for Docker on Mac, https://hub.docker.com/_/microsoft-azure-sql-edge. And I couldn't find a way to get it to work on my machine either.

Best guess, it's stored inside the container, so if it's ever destroyed the data will be lost.

Reply:
Fair enough, good things it's just local dev. Thanks Oscar!